oaknational/Oak-Web-Application
# Feat/i want a user to be geolocated at sign up - LESQ-1026

## Description

Music year: 2012

- Uses "x-country" header to set location 
- Sets DEVELOPMENT_USER_REGION  env for development (added to dev config)




## How to test

1. Clear any public meta data in clerk
2. Go to https://deploy-preview-2720--oak-web-application.netlify.thenational.academy/onboarding
3. Click through onboarding steps
4. Check clerk meta data - It should show "region" data
`{ "owa": { "isTeacher": false, "isOnboarded": true }, "region": "GB", "sourceApp": "https://deploy-preview-2720--oak-web-application.netlify.thenational.academy" }`

5. Clear public meta data in clerk
6. Use proxy to use different location
7. repeat steps above - you should see different country code
`{ "owa": { "isTeacher": false, "isOnboarded": true }, "region": "US", "sourceApp": "https://deploy-preview-2720--oak-web-application.netlify.thenational.academy" }`

8. disable proxy and logout/login - original signup region should still exist


## Checklist
